prow/external-plugins/cherrypicker/server.go

```diff
@@ -256,6 +257,8 @@ func (s *Server) handlePullRequest(l *logrus.Entry, pre github.PullRequestEvent)
 
 	// requestor -> target branch -> issue comment
 	requestorToComments := make(map[string]map[string]*github.IssueComment)
+
+	// first look for magic comments
```

Comment:
not super useful as a comment -- these are not really magic, just how the plugin works